api/api_test.go

```diff
@@ -210,21 +210,26 @@ func TestGetPluginsWithEmptyPluginRegistry(t *testing.T) {
 
 func TestPools(t *testing.T) {
 	api := API{
-		Pools: map[string]*pool.Pool{
-			config.Default: pool.NewPool(context.TODO(), config.EmptyPoolCapacity),
+		Pools: map[string]map[string]*pool.Pool{
+			config.Default: {config.DefaultConfigurationBlock: pool.NewPool(context.TODO(), config.EmptyPoolCapacity)},
 		},
 		ctx: context.Background(),
 	}
 	pools, err := api.GetPools(context.Background(), &emptypb.Empty{})
 	require.NoError(t, err)
 	assert.NotEmpty(t, pools)
 	assert.NotEmpty(t, pools.AsMap())
-	assert.Equal(t, pools.AsMap()[config.Default], map[string]interface{}{"cap": 0.0, "size": 0.0})
+
+	assert.Equal(t,
+		map[string]interface{}{
```

Comment:
Nit: the `interface{}` can be replaced with `any` everywhere.

Reply:
Replaced interface{} with any for all references in this PR